N64DeltaCore/Bridge/N64EmulatorBridge.m

```diff
@@ -158,11 +158,11 @@ static void MupenInitiateControllers (CONTROL_INFO ControlInfo)
 {
     ControlInfo.Controls[0].Present = 1;
     ControlInfo.Controls[0].Plugin = PLUGIN_RAW;
```

Comment:
Unrelated to this change, but I separately changed this value to `PLUGIN_MEMPAK` on my own branch in the meantime so it should be switched for consistency.

```suggestion
    ControlInfo.Controls[0].Plugin = PLUGIN_MEMPAK;
```